README.md

```diff
@@ -151,6 +151,36 @@ type NftMetadata = {
 
 See the implementation of the [Ethers](https://github.com/spectrexyz/use-nft/blob/38bd803f20e778b9bb684d682c194a812a94a05c/src/fetchers/ethers/index.tsx#L12-L42) and [Ethereum](https://github.com/spectrexyz/use-nft/blob/38bd803f20e778b9bb684d682c194a812a94a05c/src/fetchers/ethereum/index.tsx#L12-L42) fetchers for more details.
 
+## Angular support
+
+To add support for this library, first install `core-js`
+
+```
+npm i -s core-js
+```
+
+then add the following line to `src/polyfills.ts`
+
+```
+import 'core-js/features/promise';
+```
+
+To use this library, import the `NftProviderAngular` class.
+
+```
+import { NftProviderAngular } from 'use-nft'
+```
+
+pass the fetcher declaration to the nftProvider and call the useNft function to retreive the NFT data.
+
+```
+const fetcher: FetcherDeclaration = ['ethers', { ethers, provider: ethers.getDefaultProvider() }];
+const nftProvider = new NftProviderAngular(fetcher);
+const result: NftMetadata = await nftProvider.useNft('0xd07dc4262bcdbf85190c01c996b4c06a461d2430', '90473');
```

Comment:
I try to only use JS for the README examples, except when explicitly refering to types (so that developers don’t have to know TS to read the examples). I think we could do the same here, and remove the types?